Package.swift

```diff
@@ -1,24 +1,11 @@
 import PackageDescription
 
-let excludes: [String]
-#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
-    excludes = [
-        "Sources/Deprecations+Removals.swift",
-        "Sources/ObjectiveCBridging.swift"
-    ]
-#else
-    excludes = [
-        "Sources/Deprecations+Removals.swift",
-        "Sources/DynamicProperty.swift",
-        "Sources/NSObject+KeyValueObserving.swift",
-        "Sources/ObjectiveCBridging.swift"
-    ]
-#endif
-
 let package = Package(
-    name: "ReactiveCocoa",
+    name: "ReactiveSwift",
```

Comment:
1. I'm not sure if a package for ReactiveCocoa is useful
2. I couldn't find how to make write a `Package.swift` to make multiple packages or use intra-repository dependencies

If this is something we want, I think we should add it back when ReactiveSwift is moved to its own repo.